sematic/scheduling/job_scheduler.py

```diff
@@ -154,6 +155,19 @@ def _assert_resolution_is_scheduleable(resolution: Resolution):
         raise StateNotSchedulable(
             f"The resolution {resolution.root_id} had no docker image URI"
         )
+    if (
+        resolution.client_version
+        and string_version_to_tuple(resolution.client_version)
+        < MIN_CLIENT_SERVER_SUPPORTS
```

Comment:
[Cosmetic] Please consider extracting this condition to a function.

Reply:
changed to setting a local var `valid_client_version` and checking that